tests/CSharpRestApiClientKeystone.Analyzers.Test/DataSource/ReadAsStringInUseUnitTests/StringInUseResponseAsync/Source/Class1.csx

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+
+namespace ConsoleApplication1
+{
+    class TypeName
+    {
+        public async Task<object> DeserializeAsClassWithStringAndReflectionAsync()
+        {
+            using var responseMessage = new HttpResponseMessage();
+            string strRepresentation = await responseMessage.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
use "var" instead of string to keep consistency.

Reply:
oh no, this is the test class, its expected to be "string" and the analyzer / code fix will change this to "using var", so, its like that as a purpose to validate the code fix.

Reply:
Ah, of course. Haven' had my coffee yet - yep. All good. I guess you can call me the human "fix"